release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -431,6 +431,13 @@ With this release, you can install {product-title} on Nutanix Cloud Clusters (NC
 
 For more information, see xref:../installing/installing_nutanix/preparing-to-install-on-nutanix.adoc#installation-nutanix-installer-infrastructure-reqs_preparing-to-install-on-nutanix[Infrastructure requirements].
 
+[id="ocp-4-18-GCP-custom-dns_{context}"]
+==== Installing a cluster on Google Cloud Platform (GCP) with customer managed DNS
+
+You can now install a cluster on GCP with a custom DNS solution that you manage. If your organization's security policies do not allow the use of public DNS services like Google Cloud DNS, you can use your own DNS service with this feature. If you provide your own DNS solution, it must include entries for the `api.<cluster_name>.<base_domain>` and `*.apps.<cluster_name>.<base_domain>` records.
+
+For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp[Installation configuration parameters].
```

Comment:
Because this isn't the actual title of the section, not sure we need the capital I
```suggestion
For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp[installation configuration parameters].
```